src/components/ProgressIndicator/ProgressItem.js

```diff
@@ -20,6 +20,7 @@ const ProgressItem = ({completed, children}) => {
 ProgressItem.propTypes = {
   completed: PropTypes.bool.isRequired,
   children: PropTypes.node,
+  modifiers: PropTypes.array,
```

Comment:
make this explicit with all the supported modifiers please: `PropTypes.oneOf(['opt1', 'opt2'])`.

This is what the storybook documentation uses for information about possible radio choices iirc.